IdentityCore/src/util/MSIDJWTHelper.m

```diff
@@ -74,6 +76,14 @@ + (NSString *)decryptJWT:(NSData *)jwtData
     return [[NSString alloc] initWithData:bits encoding:NSUTF8StringEncoding];
 }
 
++ (NSData *)sign:(SecKeyRef)privateKey
```

Comment:
Just curious where this is being used?

Reply:
This is used for signing the jwt with private key for at pop.